dpuagent/ansible/dpuagent.py

```
@@ -0,0 +1,130 @@
+
+
+# name: install zstack-store
+if client == "false":
+    command = "bash %s %s %s" % (dest_pkg, fs_rootpath, max_capacity)
+else:
+    command = "bash " + dest_pkg
+run_remote_command(command, host_post_info)
+
+# if user is not root , Change the owner of the directory to ordinary user
+if fs_rootpath != '' and remote_user != 'root':
+    run_remote_command("sudo chown -R -H --dereference %s: %s" % (remote_user, fs_rootpath), host_post_info)
```

Ruff (0.15.2)

[error] `run_remote_command` may be undefined, or defined from star imports

(F405)